Include asciidoctor-bibtex and document README conversion

clbarnes opened the pull request:
Bibtex being, unfortunately, the closest we have to a standard for citations; and this being a one-stop-shop for document publishing, it seemed to make sense. Includes asciidoctor-bibtex v0.7.1 and a test case.

Furthermore, adds comments to resolve my own confusion around the two READMEs: see #144 and #145.

## Minor

- `make shell` recipe for running the docker file interactively
- Add links to README

dduportal commented on Makefile:
```diff
@@ -31,6 +33,9 @@ build:
 		--file=Dockerfile \
 		$(CURDIR)/
 
+run: build
```
What do you think about naming this target to `shell` instead of `run` to avoid any language misunderstanding (for non native speakers)?

clbarnes replied:
Fine by me!

clbarnes commented:
Sure! Sorry I didn't do that first time round.

dduportal commented on README.adoc:
```diff
@@ -67,8 +68,8 @@ asciidoctor-revealjs -r asciidoctor-diagram sample-slides.adoc
 ----
-asciidoctor-revealjs -a revealjsdir=https://cdnjs.cloudflare.com/ajax/libs/reveal.js/3.9.2 sample-slides.adoc
-asciidoctor-revealjs -a revealjsdir=https://cdnjs.cloudflare.com/ajax/libs/reveal.js/3.9.2 -r asciidoctor-diagram sample-slides.adoc
+asciidoctor-revealjs -a revealjsdir=https://cdnjs.cloudflare.com/ajax/libs/reveal.js/3.8.0 sample-slides.adoc
```
```suggestion
asciidoctor-revealjs -a revealjsdir=https://cdnjs.cloudflare.com/ajax/libs/reveal.js/3.9.2 sample-slides.adoc
```

dduportal replied:
Looks a minor typo due to the rebase, after #136 was merged.

dduportal commented on README.adoc:
```diff
@@ -67,8 +68,8 @@ asciidoctor-revealjs -r asciidoctor-diagram sample-slides.adoc
-asciidoctor-revealjs -a revealjsdir=https://cdnjs.cloudflare.com/ajax/libs/reveal.js/3.9.2 sample-slides.adoc
-asciidoctor-revealjs -a revealjsdir=https://cdnjs.cloudflare.com/ajax/libs/reveal.js/3.9.2 -r asciidoctor-diagram sample-slides.adoc
+asciidoctor-revealjs -a revealjsdir=https://cdnjs.cloudflare.com/ajax/libs/reveal.js/3.8.0 sample-slides.adoc
+asciidoctor-revealjs -a revealjsdir=https://cdnjs.cloudflare.com/ajax/libs/reveal.js/3.8.0 -r asciidoctor-diagram sample-slides.adoc
```
```suggestion
asciidoctor-revealjs -a revealjsdir=https://cdnjs.cloudflare.com/ajax/libs/reveal.js/3.9.2 -r asciidoctor-diagram sample-slides.adoc
```

dduportal replied:
Looks a minor typo due to the rebase, after #136 was merged.

clbarnes commented:
Thanks for spotting that - it was a messy merge, must have got lost in the shuffle. I've fixed that and also added some documentation for the next person who suggests #144 :grin:
